tests/test_unit/test_validators/test_datasets_validators.py

```diff
@@ -274,15 +259,21 @@ def test_bboxes_dataset_validator_with_invalid_shape_array(
         (
             [1, 2, 3],
             pytest.raises(ValueError),
-            "Expected 'individual_names' to have length 2, but got 3.",
-        ),  # length doesn't match position_array.shape[1]
+            "Expected 'individual_names' to have length 2, "
+            f"but got {len([1, 2, 3])}.",
+        ),
+        # length doesn't match position_array.shape[1]
```

Comment:
same comment as before, I find it more natural to have the comment before the code